clients/fossid-webapp/src/main/kotlin/PostRequestBody.kt

```diff
@@ -19,16 +19,24 @@
 
 package org.ossreviewtoolkit.clients.fossid
 
-class PostRequestBody(
+data class PostRequestBody private constructor(
```

Comment:
What's the benefit of turning this into a data class? Please provide a rationale in the commit message.

Reply:
> The generated functions may be useful.

In what context might the generated functions be useful? I mean, we'll probably never need a proper `equals` for a `PostRequestBody`.

Don't get me wrong, I'm not completely against turning this into a data class. I'm just curious about the direction behind that idea, if any. Because standing on its own, this just increases the number of code lines without an obvious benefit currently.

Reply:
I didn't have any specific case in mind. The class just seemed like a perfect fit for a data class, and I thought the client library might be used also from outside ort. Maybe some test compares a constructed test via `equals` maybe someone adds it to a `Set` / `Map` or uses `copy()` to change a single attribute value. 

I wouldn't have an issue at all with dropping this commit. What do you prefer?

Reply:
Ok, thanks for your thoughts. Let's keep the code then now that it's there.